lib/shared/addon/components/cluster-driver/driver-okecapi/component.js

```diff
@@ -15,7 +15,7 @@ export default Component.extend(ClusterDriver, {
   oci:                 service(),
   canChangeNetworking: true,
   layout,
-  configField:         'ociocneEngineConfig',
+  configField:         'okecapiEngineConfig',
```

Comment:
Does this need to be changed anywhere else besides this okecapi/component.js file? I searched for "ociocneEngineConfig" in the rancher-ui repo for comparison and didn't see any additional references, but am wondering about vz repo

Reply:
just in this file